tests/product/tests/conftest.py

```diff
@@ -1,22 +1,22 @@
 import pytest
-import time
-from commonlib.io_utils import  get_k8s_yaml_objects
+from commonlib.io_utils import *
```

Comment:
@oren-zohar, according to PEP8 - imports section, wildcard imports should be avoided.
[PEP8 Imports](https://peps.python.org/pep-0008/#imports)